src/main/java/com/kuit/agarang/domain/login/util/JwtProvider.java

```diff
@@ -0,0 +1,70 @@
+package com.kuit.agarang.domain.login.util;
+
+import com.kuit.agarang.global.common.exception.jwt.bad_request.JwtUnsupportedTokenException;
+import com.kuit.agarang.global.common.exception.jwt.unauthorized.JwtInvalidTokenException;
+import com.kuit.agarang.global.common.exception.jwt.unauthorized.JwtMalformedTokenException;
+import io.jsonwebtoken.*;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.Date;
+
+import static com.kuit.agarang.global.common.response.status.BaseExceptionResponseStatus.*;
+
+@Slf4j
+@Component
+public class JwtProvider {
+
+  @Value("${secret.jwt-secret-key}")
+  private String JWT_SECRET_KEY;
+
+  @Value("${secret.jwt-expired-in}")
+  private long JWT_EXPIRED_IN;
+
+  public String createToken(String userId) {
+    log.info("JWT key={}", JWT_SECRET_KEY);
+
+    Claims claims = Jwts.claims();
+    Date now = new Date();
+    Date validity = new Date(now.getTime() + JWT_EXPIRED_IN);
```

Comment:
```suggestion
Date validity = new Date(System.currentTimeMillis() + JWT_EXPIRED_IN);
```
이런 방법도 있어요! 취향차이인 것 같습니다!ㅎㅎ